makexpi.sh

```diff
@@ -44,14 +44,9 @@ if [ -n "$1" ] && [ "$2" != "--no-recurse" ] && [ "$1" != "--fast" ] ; then
 
   # Now escape from the horrible mess we've made
   cd ..
-	XPI_NAME="$APP_NAME-$1"
-  # In this mad recursive situation, sometimes old buggy build scripts make
-  # the xpi as ./pkg :(
-  if ! cp $SUBDIR/pkg/$XPI_NAME pkg/ ; then
-    echo Recovering from hair-raising recursion:
-    echo cp $SUBDIR/pkg pkg/$XPI_NAME
-    cp $SUBDIR/pkg pkg/$XPI_NAME
-  fi
+  XPI_NAME="$APP_NAME-$1"
+  cp $SUBDIR/pkg/$XPI_NAME.xpi pkg/
```

Comment:
Oh I see, it's a fix for a bug that was introduced in commits 6a7b8761bf1167a78049073f6452deec7bb20a6c and 55a7ab05f334ed50dfdfdd15ebc2cf8940882e04...